### PR TITLE
Sub emitter

### PR DESCRIPTION
**Pull request by IbraheemOsama**
Handling issue #844

### CODE REVIEW

File: content/babylon101/babylon101/Particles.md [hunk 1]
```diff
@@ -400,6 +402,23 @@ When calling `system.stop()` on a `GPUParticleSystem` object, you will force the
 
 To completely stop a `GPUParticleSystem`, you have to call `dispose()` on it.
 
+## Sub Emitters
+
+Start from babylonjs 3.2 you can create sub emitter, using this feature you can spawns a new ParticleSystem when the particle dies. every new spawned ParticleSystem is totally independent from the parent and could have different shapes so for example you could have a root ParticleSystem which produces cone shapes and the subEmitters could have a sphere shape or even you can provide more than a ParticleSystem with different shapes and let the engine choose randomly between them.
```
Comment (deltakosh) [inline content/babylon101/babylon101/Particles.md]: Starting

Comment (deltakosh) [reply content/babylon101/babylon101/Particles.md]: Some typos like "you can spawns". Also punctuation

File: content/babylon101/babylon101/Particles.md [hunk 2]
```diff
@@ -400,6 +402,23 @@ When calling `system.stop()` on a `GPUParticleSystem` object, you will force the
...
+
+### How to use it
+
+a new property is introduced in the ParticleSystem `subEmitters` which is accepting an array of `ParticleSystem` when the particle dies the engine select a random System from the `subEmitters` array and spawn a new system by cloning the selected ParticleSystem. you can add `subEmitters` property to the `subEmitters` so when the particles of the spawned ParticleSystem dies it will create a new Particle System and so on.
+
+The property `manualEmitCount` would be used in the subEmitters to avoid a infinite loop of creating and spawn new systems unless you're controlling the newly generated ParticleSystem using property `activeSubSystems`.
```
Comment (deltakosh) [inline content/babylon101/babylon101/Particles.md]: Not in the submitters but in the root one

**Review by deltakosh — COMMENTED** [review]
Few comments:)

File: content/babylon101/babylon101/Particles.md (hunk 2 above)
Comment (IbraheemOsama) [reply content/babylon101/babylon101/Particles.md]: No, it is used in the SubEmitters because this way the subemitters will die once it is done :)

Comment (deltakosh) [reply content/babylon101/babylon101/Particles.md]: Yeah you are right :)

**Comment by IbraheemOsama** [comment]
@deltakosh any other comments :D ?

**Comment by deltakosh** [comment]
Nope! thanks a lot

**Pull request merged**